specs/common/error-signalling.md

```diff
@@ -0,0 +1,155 @@
+# Error Signalling
+
+<!-- copied from ../serving/knative-api-specification-1.0.md#error-signalling -->
+
+Knative APIs use the
+[Kubernetes Conditions convention](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties)
+to communicate errors and problems to the user. Note that Knative customizes the
+general Kubernetes recommendation with a `severity` field, and does not include
+`lastTransitionTime` for scalability reasons. Each user-visible resource
+described in Resource Overview MUST have a `conditions` field in `status`, which
+MUST be a list of `Condition` objects of the following form. Fields in the
+condition which are not marked as REQUIRED may be omitted to indicate the
```

Comment:
I think we need some text here, and possibly other docs that have tables like this, to talk about what REQUIRED and OPTIONAL means in tables like this. For example, looking at:
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/1944671/121807036-6e828780-cc20-11eb-9a51-3a8e1c5d1150.png">
do we ever say that OPTIONAL in the 2nd column means that it's OPTIONAL for a user to specify it but it's still REQUIRED for an impl to support it? May seem obvious to us, but I think we need to spell it out.